src/renderer/canvas/label.ts

```diff
@@ -25,6 +32,13 @@ export interface ILabelData {
   properties: Partial<ILabelProperties>;
 }
 
+type BorderPoint = {
```

Comment:
Btw, we do all interfaces and types with the prefix `I` so we can differentiate it from the classes with the same name.

Reply:
But, you already have a type you need, it is called `IRectangle` from `common/rectangle.ts`. Same fields, same types.